src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/GroupsController.kt

```diff
@@ -73,6 +73,46 @@ class GroupsController(
     return groupsService.getGroupDetail(group)
   }
 
+  @GetMapping("/groups/child/{group}")
+  @PreAuthorize("hasRole('ROLE_MAINTAIN_OAUTH_USERS')")
+  @Operation(
+    summary = "Child Group detail.",
+    description = "get Child Group Details"
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "OK"
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "Child Group not found.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      )
+    ]
+  )
+  fun getChildGroupDetail(
```

Comment:
General comment - not just yours - should we also have a 403 in the swagger? Maybe discuss as team.